test/WebJobs.Script.Tests.Integration/Management/InstanceManagerTests.cs

```diff
@@ -731,6 +731,76 @@ static async void verifyProperties(HttpRequestMessage request, CancellationToken
                 p => Assert.StartsWith("Specialize MSI sidecar returned OK", p));
         }
 
+        [Fact]
```

Comment:
how does this test differ from the one above? Rather than having both, can we just verify all the properties in a single test?

Reply:
Yeah we can, wasn't sure if it would be better to separate them. I can just add it within the previous test to also check the identity properties. One question I had was whether all the ManagedServiceIdentity properties are required?